README.md

```diff
@@ -26,6 +26,7 @@ Inspired by Terry A. Davis (in memoriam).
 
 ## Generating the ISO image
 
+`make check_dir`
```

Comment:
You need to create a rule in the Makefile for `check_dir`.

Reply:
not sure what you mean by creating a rule for `check_dir` if it already exists

```make
check_dir:
	if [ ! -d "$(OBJ_DIR)" ]; then \
		mkdir -p $(OBJ_DIR); \
	fi

```

Reply:
My bad, forgot about that.